pkg/agent/baker.go

```diff
@@ -807,5 +807,16 @@ func getContainerServiceFuncMap(config *datamodel.NodeBootstrappingConfiguration
 		"GetOutboundCommand": func() string {
 			return getOutBoundCmd(config, config.CloudSpecConfig)
 		},
+		"GPUDriverVersion": func() string {
+			if isStandardNCv1(profile.VMSize) {
```

Comment:
I do not like this, but I would prefer the fix to be in AB and as simple as possible for now...

Reply:
this miiiight break the RP egress tests for dedicated vhd depending on vm size...
maybe can cache both -_-